test/data-maker/data-maker.go

```diff
@@ -1,3 +1,5 @@
+// +build utils
+
 /*
 	Copyright 2019 NetFoundry, Inc.
```

Comment:
touched files should have the copyright updated to 2019-2020

Reply:
looking forward to some automation around this :)

Reply:
```plorenz@carrot:~/work/nf/ziti-cmd$ cat ~/bin/update-copyright 
git diff --name-only origin/master | grep "\.go$" | xargs -n 1 -I {} sed -i 's/Copyright 2019 NetFoundry/Copyright 2020 NetFoundry/g' {}```